### PR TITLE
Test BreachAlertsManager.loadBreaches() and compareToBreaches()

### PR DESCRIPTION
**Pull request by vphong**
#6705

### CODE REVIEW

File: ClientTests/BreachAlertsTests.swift [hunk 1]
```diff
@@ -0,0 +1,83 @@
+//
+//  BreachAlertsTests.swift
+//  ClientTests
+//
+//  Created by Vanna Phong on 5/29/20.
+//  Copyright © 2020 Mozilla. All rights reserved.
+//
```
Comment (garvankeeley) [inline ClientTests/BreachAlertsTests.swift:7]: Please use the standard Mozilla license header (copy from another file)

Comment (vphong) [reply ClientTests/BreachAlertsTests.swift]: Changed across all files I've created. Thanks for pointing that out!

Comment (kaylagalway) [reply ClientTests/BreachAlertsTests.swift]: Ah thanks garvan, missed that!

File: Client/Frontend/Login Management/BreachAlertsManager.swift [hunk 2]
```diff
@@ -44,8 +40,7 @@ final public class BreachAlertsManager {
 
         self.breachAlertsClient.fetchData(endpoint: .breachedAccounts) { maybeData in
             if maybeData.isSuccess, let data = maybeData.successValue {
-                let decoder = JSONDecoder()
-                if let decoded = try? decoder.decode([BreachRecord].self, from: data) {
+                if let decoded = try? JSONDecoder().decode([BreachRecord].self, from: data) {
                     self.breaches = decoded
                     completion(Maybe(success: self.breaches))
                 }
```
Comment (kaylagalway) [inline Client/Frontend/Login Management/BreachAlertsManager.swift:46]: Just realized there should be an else here if it fails to decode

File: Client/Frontend/Login Management/BreachAlertsManager.swift [hunk 3]
```diff
@@ -44,11 +40,13 @@ final public class BreachAlertsManager {
 
         self.breachAlertsClient.fetchData(endpoint: .breachedAccounts) { maybeData in
             if maybeData.isSuccess, let data = maybeData.successValue {
-                let decoder = JSONDecoder()
-                if let decoded = try? decoder.decode([BreachRecord].self, from: data) {
-                    self.breaches = decoded
-                    completion(Maybe(success: self.breaches))
+                guard let decoded = try? JSONDecoder().decode([BreachRecord].self, from: data) else {
```
Comment (garvankeeley) [inline Client/Frontend/Login Management/BreachAlertsManager.swift:43]: Prefer `guard let data = maybe.successValue else { ... }` on line 42. 
Do you need to check maybe.isSuccess? I think that can be left out

Comment (vphong) [reply Client/Frontend/Login Management/BreachAlertsManager.swift:43]: No, not needed. Will change to `guard` syntax in next commit

**Pull request merged**